source/common/quic/active_quic_listener.cc

```diff
@@ -165,7 +169,9 @@ void ActiveQuicListener::pauseListening() { quic_dispatcher_->StopAcceptingNewCo
 
 void ActiveQuicListener::resumeListening() { quic_dispatcher_->StartAcceptingNewConnections(); }
 
-void ActiveQuicListener::shutdownListener() {
+void ActiveQuicListener::shutdownListener(Network::ExtraShutdownListenerOptionsSharedPtr options) {
+  hot_restart_udp_packet_forwarding_ =
+      std::dynamic_pointer_cast<Network::HotRestartUdpPacketForwarding>(options);
```

Comment:
In some Envoy build environments (where RTTI is disabled) I think this will not work :/

Reply:
Changed.